walkthroughs/3a-low-code-api-development/walkthrough.adoc

```diff
@@ -189,26 +235,46 @@ A *Path* represents an API endpoint/operation and the associated parameters requ
 
 . Click the orange *POST* icon.
 
-. Enter "Greet with name" in the *Summary* field.
+. Enter the following in the *Summary* field:
++
+----
+Greet with name
+----
 
-. Enter "greetname" in the *Operation ID* field.
+. Enter the following in the *Operation ID* field:
++
+----
+greetname
+----
 
-. In the *Request Body* section, choose the *Name* type that you created earlier.
+. In the *Request Body* click *Add a request body* and choose the *Name* type that you created earlier.
 
 . In the *Responses* section click *Add response*.
-* In the modal that appears choose *200 OK* as the dropdown option.
-* Click *Add*.
-* Click *No description* beside the *200 OK* response and enter "Greeting response" in the *Description* field.
+.. In the modal that appears choose *200 OK* as the dropdown option.
+.. Click *Add*.
+.. Click *No description* beside the *200 OK* response and enter the following in the *Description* field:
```

Comment:
We  need to add a line "Set the `Response Type` to `String as String`" before the Save step.

Reply:
Correction. It can be the `Name` type we defined earlier.

Reply:
outdated